test/WebSites/BasicWebSite/Areas/Area1/Views/RemoteAttribute_Home/Details.cshtml

```diff
@@ -64,13 +61,5 @@
 </div>
 
 <p>
-    @Html.ActionLink("Create a new one", "Create") |
-    @if (string.Equals("Aria", areaName, StringComparison.OrdinalIgnoreCase))
-    {
-        @Html.RouteLink("Go back to home", "Home")
-    }
-    else
-    {
-        @Html.RouteLink("Go to Aria home", "AriaHome")
-    }
+    @Html.ActionLink("Create a new one", "Create") | @Html.RouteLink("Go to Area1 home", "AriaHome")
```

Comment:
Previously areas and non-area based controllers used the views in the root shared folder. But I felt that since these files are being moved to BasicWebSite, their structure doesn't make sense and also this is not a pivot here. This kind of scenario should have been covered in 'area' specific tests? 
Regarding RemoteAttribute and areas, the `DisplayFor` extensions on the individual properties of the model generate links and they should still behave as expected?